lms/models/oauth2_token.py

```diff
@@ -0,0 +1,60 @@
+"""Database model for persisting OAuth 2 tokens."""
+import datetime
+
+import sqlalchemy as sa
+
+from lms.db import BASE
+
+
+class OAuth2Token(BASE):
+    """An OAuth 2.0 access token, refresh token and expiry time."""
```

Comment:
It would be useful here to have a brief note about what OAuth 2 tokens are used for in the context of this app, especially since we have quite a lot of different kinds of tokens and keys being used for various purposes in the lms app.

Reply:
Added